dataland-frontend/src/components/pages/MyDataRequestsOverview.vue

```diff
@@ -0,0 +1,486 @@
+<template>
+  <AuthenticationWrapper>
+    <TheHeader />
+    <DatasetsTabMenu :initial-tab-index="3">
+      <TheContent class="min-h-screen paper-section relative">
+        <div v-if="!waitingForData && storedDataRequests.length > 0">
+          <div
+            id="searchBarAndFiltersContainer"
+            class="w-full bg-white pt-4 justify-between"
+            ref="searchBarAndFiltersContainer"
+          >
+            <span class="align-content-start flex items-center justify-start">
+              <span class="w-3 p-input-icon-left" style="margin: 15px">
+                <i class="pi pi-search pl-3 pr-3" aria-hidden="true" style="color: #958d7c" />
+                <InputText
+                  data-test="requested-Datasets-searchbar"
+                  v-model="searchBarInput"
+                  placeholder="Search by company name"
+                  class="w-12 pl-6 pr-6"
+                />
+                <i class="pi pi-search pl-3 pr-3" aria-hidden="true"></i>
+              </span>
+              <FrameworkDataSearchDropdownFilter
+                v-model="selectedFrameworks"
+                ref="frameworkFilter"
+                :available-items="availableFrameworks"
+                filter-name="Framework"
+                data-test="requested-Datasets-frameworks"
+                filter-id="framework-filter"
+                filter-placeholder="Search frameworks"
+                class="ml-3"
+                style="margin: 15px"
+              />
+              <div class="flex align-items-center">
+                <span
+                  data-test="reset-filter"
+                  style="margin: 15px"
+                  class="ml-3 cursor-pointer text-primary font-semibold d-letters"
+                  @click="resetFilterAndSearchBar"
+                  >RESET</span
+                >
+              </div>
+            </span>
+          </div>
+          <div class="col-12 text-left p-3">
+            <div class="card">
+              <DataTable
+                :value="displayedData"
+                class="table-cursor"
+                :rowHover="true"
+                data-test="requested-Datasets-table"
+                paginator
+                paginator-position="bottom"
+                :rows="datasetsPerPage"
+                lazy
+                :total-records="numberOfFilteredRequests"
+                @page="onPage($event)"
+                @sort="onSort($event)"
```

Comment:
aktuell ist die row nicht klickable.
hier kannst Du die row clickable machen mit 
`@row-click="goToResolveDataRequestPage(...)`
dafür müsstest du goToResolveDataRequestPage() anpassen zu sowas wie ifClickableGoToResolveDataRequestPage()

Reply:
https://primevue.org/datatable/#api.datatable.emits

Reply:
was hälst Du davon?

Reply:
"Resolve button is clickable, rest is not. Resolve button takes you to the view single dataset page that matches the request", hatte das so verstanden, dass die row nicht clickable sein sollte?